### ISSUE TITLE
Blurry icon after animation

### ISSUE THREAD

**Issue by fikrirasyid** [elegantthemes/Divi-Beta#764]
### Problem:
Reported by @timayhelen. In chrome, icon becomes blurry after being animated:

![blurry](https://cloud.githubusercontent.com/assets/916442/20331717/c21c83e4-abd9-11e6-8d04-69f1b44c8c0a.gif)

**Comment by timayhelen** [comment]
@fikrirasyid Can we make sure this only targets icon fonts? Because it's quite noticeable when opened a modal and seeing all of the text change after the animation. Also the fix needs to be applied to module icons in the "insert module" modal and add from library modal.

**Comment by fikrirasyid** [comment]
@timayhelen I've modified the way the fix works. I initially aimed to incorporate your feedback about `Can we make sure this only targets icon fonts` by comprehensively list all the icon-only selector, but later I found a better way to implement the fix. It turns out that the issue appears consistently on all fonts after its DOM applies `et_pb_fade_in_scale` transition (probably due to `scale()`). Thus the issue can be wiped once and for all by adding `font-smoothing` on `et_pb_fade_in_scale`'s 100% state.
